pkg/synchromanager/clustersynchro/resource_negotiator.go

```diff
@@ -141,7 +141,7 @@ func (negotiator *ResourceNegotiator) NegotiateSyncResources(syncResources []clu
 					if unstructured {
```

Comment:
The variable name `unstructured` is wrong, it should be `isLegacyResource`, caused by https://github.com/clusterpedia-io/clusterpedia/pull/180/files#diff-4f38697cf9947176f6479f11869c0a4f4b5c3209cd7608b95a71dc94b714e0f9R62

@PiotrLewandowski323 Could you please fix it?

Reply:
Yes, I've created PR #224